cloud/envs/gcp.py

```diff
@@ -24,8 +24,8 @@ def __init__(self, collect_existing_tpus=True, **kwargs):
     # Check for dependencies
     try:
       utils.call(["gcloud", "--version"])
-    except:
-      raise Exception("Missing commandline utility: gcloud")
+    except Exception as e:
+      print(e)
```

Comment:
I think we spoke over person where I said `print` was okay, now that I think about it I think we should raise it. What do you think @mathemakitten @aidangomez?